Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -9,7 +9,7 @@ The Mixed Reality Toolkit (MRTK) is a cross-platform toolkit for building Mixed
 To get started with the Mixed Reality Toolkit you will need:
 
 * [Visual Studio 2017](http://dev.windows.com/downloads)
```

Comment:
VS 2019? I think that page defaults to 2019 now, and I've heard it can be difficult / not possible? to find 2017.

Reply:
good point!

Reply:
there does not appear to be a unique page for 2017... you have to click on "older versions"

Reply:
updated download link